src/validations/rules/ssp.sch

```diff
@@ -2734,8 +2733,8 @@
                 id="implemented-requirement-has-allowed-control-origination"
                 role="error"
                 see="Guide to OSCAL-based FedRAMP System Security Plans §5.3.1.1"
-                test="oscal:prop[@ns eq 'https://fedramp.gov/ns/oscal' and @name eq 'control-origination' and @value = $control-originations]"> Every
-                implemented requirement has an allowed control origin.</sch:assert>
+                test="oscal:prop[@name eq 'control-origination' and @value = $control-originations]"> Every implemented requirement has an allowed
+                control origin.</sch:assert>
```

Comment:
In this one I would say stick to the script and maybe we call them control origination.

<img width="883" alt="Screen Shot 2021-08-13 at 3 08 01 PM" src="https://user-images.githubusercontent.com/61464190/129407318-9aa56c1f-3619-41d5-99d9-ec3881688a44.png">


```suggestion
                test="oscal:prop[@name eq 'control-origination' and @value = $control-originations]"> Every implemented requirement has an allowed
                control origination.</sch:assert>
```

Reply:
I changed two occurrences of origin to origination.